sale_procurement_group_by_line/README.rst

```diff
@@ -21,44 +22,54 @@ Configuration
 
 No special configuration is required.
 
+
```

Comment:
Why this blank line? Also I'd delete the Installation and Configuration sections, as they are empty